testing/conftest.py

```diff
@@ -1,6 +1,21 @@
 # -*- coding: utf-8 -*-
+import sys
+
 import pytest
 
+if sys.gettrace():
```

Comment:
hmmm I would have put this in the function `and orig_trace is not None`

Reply:
Well, this way it avoids using the fixture when not run with coverage (i.e. no initial tracing).
(not sure I understand your suggestion though)

Reply:
I agree it seems better to not have this active without need.